src/components/Admin/UsersTable/index.js

```diff
@@ -1,24 +1,85 @@
 import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
+import {
+  FormControl,
+  Container,
+  Row,
+  Col,
+  InputGroup,
+  DropdownButton,
+  Dropdown,
+} from 'react-bootstrap';
 import { LoadableButton } from '../../Loadable';
 import Roster from '../../Roster';
 import UserRow from './UserRow';
 
 const FloatingButton = styled(LoadableButton)`
-  right: 0;
-  top: -4em;
   padding: 1em;
 `;
 
+const ConteinerNotFound = styled.div`
+  display: ${(prop) => (!prop.empty ? 'flex' : 'none')};
+  justify-content: center;
+`;
+
+const MessageNotFound = styled.p`
+  font-size: 3em;
+`;
+
 const UsersTable = () => {
   const [selectedUsers, setSelectedUsers] = useState([]);
   const [users, setUsers] = useState([]);
   const [loading, setLoading] = useState(false);
+  const [filter, setFilter] = useState('');
+  const [serch, setSerch] = useState('Nombre');
+
+  const TITLES_TABLE_USER = [
+    'Selección',
+    '#',
+    'Mail',
+    'Password',
+    'Rol',
+    'Permisos',
+    'Acciones',
+  ];
 
   useEffect(() => {
     upDateTable();
   }, []);
 
+  const handleFilterUsers = (textSerch) => {
+    if (textSerch.length === 0) {
+      upDateTable();
```

Comment:
updateTable

Reply:
upDateTable to updateTable has been replaced in all tables